Icon/style.css

```diff
@@ -1,13 +1,108 @@
 @import '../variables.css';
 
+.curious-blue {
```

Comment:
just a thought here. what if these were renamed to the JS equivalents `curiousBlue`.

Then you could just add the `color` prop to to classNames like this

```js
const classes = classNames(styles, 'icon', color);
```

The proptypes would ensure that you pass in a valid color 😄